src/command_helpers/asset_cache.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Metalware
+  module CommandHelpers
+    module AssetCache
+      private
```

Comment:
So the mixin doesn't need the `private`. If the user wishes to make the method provide, they can include it as such in the calling class.